Sweep_Obstacle_Detector/Sweep_Obstacle_Detector.ino

```diff
@@ -1,53 +1,55 @@
-const uint16_t brakeTime_ms           = 1000;   // Time needed to fully stop after applying brake
+const uint8_t POWER_PIN = 2; // Power Enable pin
+const uint8_t SYNC_PIN = 3; // Sync / Device Ready pin
+const uint8_t BRAKE_PIN = 4; // Brake pin
+const uint8_t SOFTSERIAL_TX_PIN = 9; // SoftwareSerial output
+const uint8_t SOFTSERIAL_RX_PIN = 8; // SoftwareSerial input
+
+const uint32_t SERIAL1_BAUD_RATE = 115200; // Baud rate for Serial1
+const uint32_t SERIAL_BAUD_RATE = 9600; // Baud rate for Serial
+const uint8_t SERIAL_TIMEOUT_MS = 5; // Timeout for serial communication
+const uint16_t READING_DELAY_US = 500; // Delay between readings
+
+const uint16_t vehicleSpeed_cmPs = 100; // Speed of vehicle (centimeters per second)
+const uint16_t brakeTime_ms = 1000; // Time needed to fully stop after applying brake
 const uint16_t RANGE_CM = ((brakeTime_ms / 1000) * vehicleSpeed_cmPs) * 2; // Range within which to scan for obstacles
-const uint16_t VEHICLE_WIDTH_CM       = 76;     // Width of vehicle (in centimeters)
+const uint16_t VEHICLE_WIDTH_CM = 76; // Width of vehicle
 
-const uint8_t  (&MOTOR_SPEED_CODE)[2] = MOTOR_SPEED_CODE_8_HZ;   // Number of rotations per second
-const uint8_t  (&SAMPLE_RATE_CODE)[2] = SAMPLE_RATE_CODE_500_HZ; // Number of samples per second
+const uint8_t MOTOR_SPEED_HZ = 8;   // Number of rotations per second
+const uint8_t (&MOTOR_SPEED_CODE)[2] = MOTOR_SPEED_CODE_8_HZ;   // Number of rotations per second
+const uint16_t SAMPLE_RATE_HZ = 500; // Number of samples per second
+const uint8_t (&SAMPLE_RATE_CODE)[2] = SAMPLE_RATE_CODE_500_HZ; // Number of samples per second
 
-const uint16_t MAX_ANGLE              = 359; // Maximum whole angle value
-const uint16_t RIGHT_ANGLE_BOUNDARY   = 90;  // Right boundary angle of scan range
-const uint16_t LEFT_ANGLE_BOUNDARY    = 270; // Left boundary angle of scan range
-const uint8_t  BRAKE_PIN_TIME_MS      = 100; // Length of time to hold brake pin high
-const uint16_t MIN_DISTANCE_CM        = 2;   // Minimum distance for reading to be valid
-const uint16_t MAX_ANGLE_DIFF_DEGREES = 30;  // Maximum angle difference between readings to be considered valid reading
-const uint8_t  MIN_SIG_STRENGTH       = 100; // Minimum signal strength for obstacle reading to be valid
+const uint16_t MAX_ANGLE = 359; // Maximum whole angle value
+const uint16_t RIGHT_ANGLE_BOUNDARY = 90; // Right boundary angle of scan range
+const uint16_t LEFT_ANGLE_BOUNDARY = 270; // Left boundary angle of scan range
+const uint8_t BRAKE_PIN_TIME_MS = 100; // Length of time to hold brake pin high
+const uint16_t MIN_DISTANCE_CM = 2; // Minimum distance for reading to be valid
+const uint16_t MAX_ANGLE_DIFF_DEGREES = 30; // Maximum angle difference between readings to be considered valid reading
+const uint8_t MIN_SIG_STRENGTH = 100; // Minimum signal strength for obstacle reading to be valid
 
 const uint8_t NUM_FRACTIONAL_BITS = 4; // Number of fractional bits in raw angle
```

Comment:
The constant values above can be preprocessor #define constants.  That way they won't take up storage.  (Some C/C++ compilers might "optimize away" constant variables, and there are optimization flags that can be modified for Arduino, but why make it do that?)  Another thing that can be done is to move them to a header file, if they need to be used by anything else.  If they are your own definitions, they could go in Settings.h.  Some changes might be needed where these are used, once they are preprocessor constants.

Since we're about to refactor the repo, and this has been patiently waiting, let's go with it as it stands, and make the changes afterward.